content/docs/start/experiments.md

```diff
@@ -37,12 +110,13 @@ $ dvc exp run --set-param featurize.max_features=3000
 running experiments. The `--set-param` (or `-S`) flag sets the values for
 <abbr>parameters<abbr> as a shortcut for editing `params.yaml`.
```

Comment:
This whole paragraph repeats what you have already explained above.

Reply:
This is removed in #2574.